client/client-lib/src/api.rs

```diff
@@ -828,7 +828,7 @@ pub struct PeerResponse<R> {
 }
 
 impl<C: JsonRpcClient> FederationMember<C> {
-    #[instrument(fields(peer = %self.peer_id), skip_all)]
+    #[instrument(level = "trace", fields(peer = %self.peer_id), skip_all)]
```

Comment:
Since we are at it, what makes that log line especially annoying is not knowing anything about the request sent. Logging the `method` field would be very useful imo.